lib/auth/auth_test.go

```diff
@@ -2701,6 +2702,25 @@ func TestCAGeneration(t *testing.T) {
 	}
 }
 
+func TestGetLicense(t *testing.T) {
+	suite := newAuthSuite(t)
+
+	// GetLicense should return error if license is not set
+	_, err := suite.a.GetLicense(context.Background())
+	assert.NotNil(t, err)
+
+	// GetLicense should return cert and key pem concatenated, when license is set
+	l := license.License{
+		CertPEM: []byte("cert"),
+		KeyPEM:  []byte("key"),
+	}
+	suite.a.SetLicense(&l)
+
+	license, err := suite.a.GetLicense(context.Background())
```

Comment:
I would rename this to `actual` or something to not conflict with imports:
`Variable 'license' collides with imported package name `